src/Akka.Persistence.Sql.Linq2Db/Config/TagTableColumnNames.cs

```diff
@@ -0,0 +1,19 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="TagTableColumnNames.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2022 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+
+namespace Akka.Persistence.Sql.Linq2Db.Config;
+
+public sealed class TagTableColumnNames
```

Comment:
Should this be here? looks premature.

Reply:
Yeah, those snuck in from another PR. I'll remove those.

Reply:
So we need to remove this - were there other compat changes that needed to be introduced to this PR to address the rest of @to11mtm 's concerns?